### PR TITLE
Fix to_remote output redeemscript when anchors enabled

### PR DESCRIPTION
**Pull request by ksedgwic**

### CODE REVIEW

File: lightning/src/ln/chan_utils.rs [hunk 1]
```diff
@@ -1129,16 +1140,20 @@ impl CommitmentTransaction {
...
-			let script = script_for_p2wpkh(&countersignatory_pubkeys.payment_point);
-			txouts.push((
-				TxOut {
-					script_pubkey: script.clone(),
-					value: to_countersignatory_value_sat,
-				},
-				None,
-			))
-		}
+                if to_countersignatory_value_sat > 0 {
```
Comment (devrandom) [inline lightning/src/ln/chan_utils.rs]: indentation

File: lightning/src/ln/chan_utils.rs [hunk 2]
```diff
@@ -608,6 +608,17 @@ pub fn build_htlc_transaction(commitment_txid: &Txid, feerate_per_kw: u32, conte
 	}
 }
 
+/// Gets the witnessScript for the to_remote output when anchors are enabled.
+#[inline]
+pub(crate) fn get_to_remote_with_anchors_redeemscript(payment_point: &PublicKey) -> Script {
```
Comment (devrandom) [inline lightning/src/ln/chan_utils.rs]: to_countersignatory

**Review by devrandom — COMMENTED** [review]
probably good to add something to the `test_anchors` test

File: lightning/src/ln/chan_utils.rs (hunk 2 above)
Comment (ksedgwic) [reply lightning/src/ln/chan_utils.rs]: fixed

File: lightning/src/ln/chan_utils.rs (hunk 1 above)
Comment (ksedgwic) [reply lightning/src/ln/chan_utils.rs]: fixed

**Pull request merged**